bciers/apps/reporting/src/app/components/taskList/headerElementFactory.ts

```diff
@@ -16,7 +16,9 @@ export const headerElementFactories: {
   }),
   [HeaderStep.ReportInformation]: (context) => ({
     type: "Section",
-    title: `${context?.facilityName ?? "Facility"} information`,
+    title: context?.isElectricityImport
+      ? "Report Information"
+      : `${context?.facilityName ?? "Facility"} information`,
```

Comment:
Should we just create another `HeaderStep` entry for EIOs instead of adding conditionals to conditionals? It would simplify everything